src/pages/Competitions.svelte

```diff
@@ -9,15 +9,58 @@ import PageHeader from "../components/PageHeader.svelte";
     <div class="content">
         <div class="content-inner">
             <h2>CS:GO Wingman 2v2</h2>
-            <p>Mer informasjon kommer</p>
+            <p>Wingman mode i CS består av runder der spillere møtes 2v2 i duell. Totalt 16 runder pr match.
+            4 lag totalt kan melde seg på. (OBS: Husk lagnavn!)
+            Turneringen vil bli hostet via Challengermode, så konto der er påkrevd og kobling mot Steam er påkrevd.
+            Lag kaptein må opprette team. Navnet på teamet må matche det som står i infoskjema.
+                Premier:
+                1. 700kr + billett til neste LAN
```

Comment:
Du må bruke ol og li for ordered lists. Se her: https://www.w3schools.com/html/html_lists_ordered.asp